src/components/cards/card-booking/index.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { Button } from '../../button/button';
+import { BodySecondary } from '../../typography/body/body2';
+import { Caption } from '../../typography/caption/caption';
+import { Label } from '../../typography/label/label';
+import { Price } from '../../typography/price/price';
+import { ContainerCard } from '../containerCard';
+import { Divider } from '../divider';
+import './index.css';
+import { colors } from '../../../typography/colors';
+import { constants } from '../../../typography';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faCircleCheck, faHeart as heartRegular } from '@fortawesome/free-regular-svg-icons';
+import { faShare, faHeart as heartSolid } from '@fortawesome/free-solid-svg-icons';
+import { H3 } from '../../typography/headline/h3';
+import { formatPrice } from '../../../price-formatter';
+import { faWhatsapp } from '@fortawesome/free-brands-svg-icons';
+
+library.add(heartRegular, heartSolid);
+
+interface BookingPrices {
+  pricetotal?: number;
+  priceCondominium: number;
+  priceIptu?: number;
+  priceServiceFee?: number;
+  priceGrossRent?: number;
+}
+
+interface BookingProps {
+  nameUser: string;
+  imageUser: React.ReactNode;
+  prices: BookingPrices;
+  broker: string;
+  checkedUser: boolean;
+  type?: 'sell' | 'rent';
+}
+
+export function CardBooking({ imageUser, prices, nameUser, broker, type, checkedUser: checked }: BookingProps) {
+  const pricetotal = prices.pricetotal ? formatPrice(prices.pricetotal) : null;
+  const priceCondominium = prices.priceCondominium ? formatPrice(prices.priceCondominium) : null;
+  const priceIptu = prices.priceIptu ? formatPrice(prices.priceIptu) : null;
+  const priceServiceFee = prices.priceServiceFee ? formatPrice(prices.priceServiceFee) : null;
+  const priceGrossRent = prices.priceGrossRent ? formatPrice(prices.priceGrossRent) : null;
+  const [favorite, setFavorite] = useState<boolean>(false);
```

Comment:
Como vc ja definiu um initial value, não precisa declarar a tipagem para esses tipos de primeira ordem como `string`, `number` e `boolean`

```suggestion
  const [favorite, setFavorite] = useState(false);
```